sdk/storage/test/block_blob_client_test.cpp

```diff
@@ -41,10 +41,10 @@ namespace Azure { namespace Storage { namespace Test {
     m_blobUploadOptions.Properties.ContentLanguage = "en-US";
     m_blobUploadOptions.Properties.ContentDisposition = "attachment";
     m_blobUploadOptions.Properties.CacheControl = "no-cache";
-    m_blobUploadOptions.Properties.ContentEncoding = "gzip";
+    m_blobUploadOptions.Properties.ContentEncoding = "identity";
     m_blobUploadOptions.Properties.ContentMD5 = "";
     m_blobUploadOptions.Tier = Azure::Storage::Blobs::AccessTier::Hot;
-    m_blockBlobClient->Upload(m_blobContent, m_blobUploadOptions);
+    m_blockBlobClient->Upload(new Azure::Storage::MemoryStream(m_blobContent), m_blobUploadOptions);
```

Comment:
Memory leak here?

Reply:
No.

In the current implementation, which is not wanted by anybody, bodystream will be deleted by Azure Core.